docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Health.adoc

```diff
@@ -79,29 +99,36 @@ created by the `add-user` script. For example:
 $ curl -v --digest -u myadminuser:myadminpassword http://localhost:9990/health
 < HTTP/1.1 200 OK
 ...
-{"outcome":"UP","checks":[]}
+{"status":"UP","checks":[{"name":"boot-errors","status":"UP"},{"name":"deployments-status","status":"UP"},{"name":"server-state","status":"UP","data":{"value":"running"}}]}
 ----
 
 If the authentication fails, the  server will reply with a `401 NOT AUTHORIZED` response.
 
+=== Default Server Readiness Procedures
+
+WildFly provides some readiness procedures to test if the application server is ready to serve requests:
+
+* `boot-errors` checks that there were no errors during the server boot sequence
+* `deployments-status` checks that all deployments were deployed without errors
+* `server-state` checks that the server state is `running`
+
+The server is ready when all these procedures are `UP`.
+
+It is possible to disable them by using the MicroProfile Config property `mp.health.disable-default-procedures`.
+
 === Global Status when probes are not defined
 
-By default, the HTTP endpoints will return `UP` if there are no health check probes defined by the application.
+By default, the HTTP endpoints and management operations will return `UP` if there are no health check procedures defined by the applications deployed on WildFly.
```

Comment:
is this sentence true for management operation after changes of this RFE? /health/ready can return `DOWN` in case of failed deployment